### DISCUSSION TITLE
自定义自增组件，联动它的显示与隐藏，在提交的时候抛出  隐藏的字段 判空验证

### DISCUSSION THREAD

**Discussion by tpc-ht** [alibaba/formily#2451] [Q&A 2.x] [answered]
https://codesandbox.io/s/practical-visvesvaraya-e79cx?file=/CouponItems.tsx:416-421

![image](https://user-images.githubusercontent.com/59469126/142394516-d2a15877-abd0-4e65-98ec-831a63cf1473.png)

**Comment by janryWang** [comment, marked as answer]
basePath没拼对，最佳实践是直接修改name就行了 https://codesandbox.io/s/heuristic-currying-ykrvw?file=/CouponItems.tsx
